### PR TITLE
Added product vision for project

### PR DESCRIPTION
**Pull request by Orest-Haponov**

### CODE REVIEW

**Comment by Orest-Haponov** [comment]
@nikita-shmatko @omnomnomka  
Please check

File: documentation/projectVision.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# VISION
+ Help development teams save their projects from unexpected assembly breakdowns and security issues due to vulnerabilities in third party libraries. 
+# TARGET GROUP
+- Software developers, project managers;
+- Security/code auditors.
+# OBJECTIVES AND GOALS
+- Fast and easily find vulnerabilities in - user’s projects;
+- Have an ability to see statistical data for found vulnerabilities.
+# FEATURES
+- As a user I want to have an opportunity to register and authorization on website;
```
Comment (cx-nikita-shmatko) [inline documentation/projectVision.md]: As a user I want to have an ability to register and authorize on web application;

File: documentation/projectVision.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- As a user I want to be able to create projects on my account;
+- As a user I want to upload zip file with sources into project to analyze them on vulnerabilities;
+- As a user I want to have an opportunity to see results of analyzing uploaded sources;
```
Comment (cx-nikita-shmatko) [inline documentation/projectVision.md]: As a user I want to have an ability to see results of analyzed uploaded sources

File: documentation/projectVision.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+- As a user I want to be able to see user information of other people’s;
```
Comment (cx-nikita-shmatko) [inline documentation/projectVision.md]: @omnomnomka Not sure about this feature, what do you think?

File: documentation/projectVision.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+- As a user I want to be able to see statistic of uploaded zip files and their vulnerabilities;
+- As a user I want to have an opportunity to change my personal data.
```
Comment (cx-nikita-shmatko) [inline documentation/projectVision.md]: As a user I want to have an ability to change my personal data

File: documentation/projectVision.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+
```
Comment (cx-nikita-shmatko) [inline documentation/projectVision.md]: @omnomnomka Let's maybe add feature about tenants creation?

**Review by cx-nikita-shmatko — CHANGES_REQUESTED** [review]

File: documentation/projectVision.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# VISION
```
Comment (omnomnomka) [inline documentation/projectVision.md]: Please, add the app name at the start of document

File: documentation/projectVision.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+ Help development teams save their projects from unexpected assembly breakdowns and security issues due to vulnerabilities in third party libraries. 
+# TARGET GROUP
+- Software developers, project managers;
+- Security/code auditors.
+# OBJECTIVES AND GOALS
+- Fast and easily find vulnerabilities in - user’s projects;
+- Have an ability to see statistical data for found vulnerabilities.
+# FEATURES
+- As a user I want to have an opportunity to register and authorization on website;
+- As a user I want to be able to create projects on my account;
```
Comment (omnomnomka) [inline documentation/projectVision.md]: We need to have CRUD on projects, not just 'create' action. I think, it's better to change user story to "...I want to have an ability to create and manage several projects..."

File: documentation/projectVision.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+- As a user I want to upload zip file with sources into project to analyze them on vulnerabilities;
```
Comment (omnomnomka) [inline documentation/projectVision.md]: ZIP-file is a very detailed and technical solution for this document. Let's leave just "upload sources" and not "into project", just "upload and analyze" or "upload and scan for vulnerabilities".

File: documentation/projectVision.md (hunk 3 above)
Comment (omnomnomka) [inline documentation/projectVision.md]: Let's change to: "I want to be able to see data of scan uploaded by other users on the project available for me"

File: documentation/projectVision.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+- As a user I want to have an opportunity to see results of analyzing uploaded sources;
+- As a user I want to be able to see user information of other people’s;
+- As a user I want to be able to see statistic of uploaded zip files and their vulnerabilities;
```
Comment (omnomnomka) [inline documentation/projectVision.md]: Let's change to "I want to be able to scan project several times and see statistical data"

File: documentation/projectVision.md (hunk 5 above)
Comment (omnomnomka) [reply documentation/projectVision.md]: I don't think so. We could add it later with the scope extention if wee wlil have a time.

**Review by omnomnomka — CHANGES_REQUESTED** [review]

**Comment by Orest-Haponov** [comment]
@nikita-shmatko @omnomnomka 
Changed feature description, please check

**Review by cx-nikita-shmatko — APPROVED** [review]

**Review by omnomnomka — APPROVED** [review]

**Pull request merged**